src/main/java/io/phasetwo/service/model/jpa/entity/OrganizationsConfigEntity.java

```diff
@@ -0,0 +1,85 @@
+package io.phasetwo.service.model.jpa.entity;
+
+import jakarta.persistence.Access;
+import jakarta.persistence.AccessType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.NamedQueries;
+import jakarta.persistence.NamedQuery;
+import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
+import java.util.Objects;
+
+@NamedQueries({
+  @NamedQuery(
+      name = "getConfigsByRealm",
+      query = "SELECT config FROM OrganizationsConfigEntity config WHERE config.realmId = :realmId")
+})
+@Entity
+@Table(
+    name = "ORGANIZATIONS_CONFIG",
+    uniqueConstraints = {@UniqueConstraint(columnNames = {"REALM_ID"})})
+public class OrganizationsConfigEntity {
```

Comment:
Rather than doing our own entity that we have to update every time we make an addition, why don't we just use realm attributes under the covers with a naming convention? In other projects we use the key naming convention `_providerConfig.{extension}.{subject}.{key}`. So for this, we could use something like `_providerConfig.orgs.config.createAdminUser` and `_providerConfig.orgs.config.sharedIdps`

However, I really do like the OrganizationsConfigProvider abstraction here. Easily available on the session with great, easy to understand methods. :+1:

Reply:
Thanks Garth.

I saw the approach in 
`public static final String ORG_DEFAULT_POST_BROKER_FLOW_KEY =
      "_providerConfig.orgs.defaults.postBrokerFlow";
  public static final String ORG_DEFAULT_SYNC_MODE_KEY = "_providerConfig.orgs.defaults.syncMode";`

which are properties strictly related to the IdentityProvider and set in a IdentityProviderRepresentation object. 

The orgs global config relates to admin users(which are not a identity provider config) and sharedIdps.


But do we want to apply this approach for the global organizations config? In which _providerConfig would these two attributes fit?

Please let me know your answer @xgp . Until then I will try doing some tests.

Also I will add a small  UI modal